### DISCUSSION TITLE
Do we need the example JSON file twice?

### DISCUSSION THREAD

**Discussion by jkanner** [gwosc-tutorial/gwosc-catalog#76] [General]
It seems like the example JSON file is in two places.  Why?  Wouldn't just 1 be better?

https://github.com/gwosc-tutorial/gwosc-catalog/blob/update-docs/schema.json

https://github.com/gwosc-tutorial/gwosc-catalog/blob/update-docs/docs/schema.md

**Comment by martinberoiz** [comment]
I think we can just delete `schema.json`, which is probably outdated by now.
